icpc-wf/ansible/Makefile

```diff
@@ -21,7 +22,7 @@ $(LIBVENDOR): $(LIBVENDORTGZ)
 	-cd roles/domjudge_checkout/files && tar xzf $(notdir $<)
 endif
 
-domserver judgehost admin grafana: %: %.yml hosts group_vars/all/secret.yml $(LIBVENDOR) $(SSHKEY) $(SSHKEY).pub
+domserver judgehost admin grafana cds: %: %.yml hosts group_vars/all/secret.yml $(LIBVENDOR) $(SSHKEY) $(SSHKEY).pub
```

Comment:
For Grafana etc. we generate the certs if they dont't exist, thats not something we want for CDS right?

Reply:
Nope the only reason to put nginx in front is if you want a specific certificate, since the CDS has one on its own.